src/hooks/staticContent.ts

```diff
@@ -54,7 +60,7 @@ export const useStaticContent = <T>({
   const { data, error: queryError, loading, refetch } = useQuery(
     getQuery(type === 'json' ? QUERY_TYPES.PUBLIC_JSON_FILE : QUERY_TYPES.PUBLIC_HTML_FILE),
     {
-      variables: { name },
+      variables: { name, version: appJson.expo.version },
```

Comment:
i put it here in order to have conversations available:

i guess we missed one place:

```
const useDrawerRoutes = () => {
  const { isConnected, isMainserverUp } = useContext(NetworkContext);
  const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp });
  const { data, error, loading } = useQuery(getQuery(QUERY_TYPES.PUBLIC_JSON_FILE), {
    variables: { name: 'navigation' },
    fetchPolicy
  });
  ...
```

Reply:
and what about following parts?

```
const onPressInfoVerification = useCallback(() => {
  navigation.navigate(ScreenName.Html, {
    title: texts.screenTitles.encounterHome,
    query: QUERY_TYPES.PUBLIC_HTML_FILE,
    queryVariables: { name: 'encounter-verification' }
  });
}, [navigation]);

const onPressInfoId = useCallback(() => {
  navigation.navigate(ScreenName.Html, {
    title: texts.screenTitles.encounterHome,
    query: QUERY_TYPES.PUBLIC_HTML_FILE,
    queryVariables: { name: 'encounter-user-id' }
  });
}, [navigation]);

const onPressInfo = useCallback(() => {
  navigation.navigate(ScreenName.Html, {
    title: texts.screenTitles.encounterHome,
    query: QUERY_TYPES.PUBLIC_HTML_FILE,
    queryVariables: { name: 'encounter-verification' }
  });
}, [navigation]);

const onPressInfo = useCallback(() => {
  navigation.navigate(ScreenName.Html, {
    title: texts.screenTitles.encounterHome,
    query: QUERY_TYPES.PUBLIC_HTML_FILE,
    queryVariables: { name: 'encounter-privacy' }
  });
}, [navigation]);
```

Reply:
refactored `useDrawerRoutes` to use `useStaticContent`, so that part is now fixed as well.

as for the others: the html screen handles it internally already, so we do not need to take extra care with the navigation